app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -257,6 +270,40 @@ class Core(private val context: Context) {
         }
     }
 
+    private fun geCustomCookiePolicy(): CookiePolicy {
+            return when (context.settings().blockCookiesSelectionInCustomTrackingProtection) {
+                "all" -> CookiePolicy.ACCEPT_NONE
+                "social" -> CookiePolicy.ACCEPT_NON_TRACKERS
+                "unvisited" -> CookiePolicy.ACCEPT_VISITED
+                "third-party" -> CookiePolicy.ACCEPT_ONLY_FIRST_PARTY
+                else -> CookiePolicy.ACCEPT_NONE
+            }
+    }
+
+    private fun getCustomTrackingCategories(): Array<TrackingCategory> {
+        val categories = arrayListOf(
+            TrackingCategory.AD,
+            TrackingCategory.ANALYTICS,
+            TrackingCategory.SOCIAL,
+            TrackingCategory.MOZILLA_SOCIAL
+        )
+
+        if (context.settings().blockTrackingContentInCustomTrackingProtection) {
+            categories.add(TrackingCategory.SCRIPTS_AND_SUB_RESOURCES)
+            categories.add(TrackingCategory.CONTENT)
```

Comment:
nit: Please remove `CONTENT`, it applies a restricted list that cause a lot of breakage, we are not using this list neither on desktop nor Fenix right now.